tools/rimage/config/tgl-cavs.toml

```diff
@@ -496,3 +496,19 @@ count = 21
 	pin = [0, 0, 0xfeef, 0xf, 0xf, 0x45ff, 1, 0, 0xfeef, 0xf, 0xf, 0x1ff]
 	# mod_cfg [PAR_0 PAR_1 PAR_2 PAR_3 IS_BYTES CPS IBS OBS MOD_FLAGS CPC OBLS]
 	mod_cfg = [0, 0, 0, 0, 4096, 1000000, 128, 128, 0, 0, 0]
+
```

Comment:
any justification for enable RTNR in TGL? how about config side? 
My opinion is: TGL is already PVed, if we want to verify this, better go to mtl?

Reply:
There's quite an effort for occasional need to use to create these, so I'd like to keep it as available. My plan is to keep kconfig default n, because this is not a real usable component as mock-up version. But with manual kconfig enable this can be loaded with suitable topology to test in real DUT.

And TGL is my choice because it's the best I have available for my work. Once this is ready I can duplicate to MTL and after.

Reply:
We need to add for SW dev reasons. i.e. easier to test/fix any failure if you have HW on your desk and manifest already has the module.